style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: #ff00f2;
+}
```

Comment:
- [ ] It is a good practice to use class selector to style the content of your file rather than using element selector. Kindly reference to the class name in your `h1` element and use it to style its content.